openapi/paths/lambdas/lambdas@crypto@validate-signature.yaml

```diff
@@ -0,0 +1,31 @@
+post:
+  operationId: validateSignature
+  summary: Validated signed message 
+  tags:
+    - Lambdas
+  description: >- 
+    Given a signed message and it's authchain, validate it's authenticity.  
+    A message can be signed, for example, to prove ownership of the Entity pointers that they want to modify.
+  requestBody:
+    description: >-
```

Comment:
Can we make the body parameters have the same format as request body parameters have in `/content/denylist/{type}/{id}`?